pmd-core/src/main/java/net/sourceforge/pmd/lang/rule/xpath/internal/Visitor.java

```diff
@@ -5,8 +5,10 @@
 package net.sourceforge.pmd.lang.rule.xpath.internal;
```

Comment:
Just fyi, I think it would be nice if this class had a more descriptive name, eg `SaxonExprVisitor` or something. I'll change the name when merging unless you disagree

Reply:
Sounds good, I'll rename it.